OracleSOASuite/dockerfiles/12.2.1.3/README.md

```diff
@@ -0,0 +1,169 @@
+
+At the end of this configuration there will be 3 containers running : 1) DB Container 2) WLS Admin Server Container 3) WLS Managed Server (SOA Server) Container.
+The containers will be connected using a Docker User Defined network 
+
+
+## Create a User Defined network
+
+
+In this configuration creation of a user defined network will enable the communication between the containers just using container names. For this setup we will use a user defined network using bridge driver.
+
+Create a user defined network using the bridge driver by executing the following command :
+
+    $ docker network create -d bridge <some name>
+   
+Sample command:
+
+    $ docker network create -d bridge SOANet
+
+
+# Mount a host directory as a data volume
+
+Data volumes are designed to persist data, independent of the container’s lifecycle. The default location of the volume in container is under `/var/lib/docker/volumes`. There is an option to mount a directory from the host into a container as volume. In this project we will use that option for the data volume. The volume will be used to store Database datafiles and WebLogic server domain files.This volume will be created on the host at `/scratch/DockerVolume/SOAVolume/`. Since the volume is created as "root" user, provide read/write/execute permissions to "oracle" user (by providing permissions to "others"), as all operations inside the container happens with "oracle" user login.
+
+To determine if a user already exists on your host system with uid:gid of 1000, run:
+
+    # getent passwd 1000
+
+If that returns a username (which is the first field), use that user for the `useradd` command below. If not, create the `oracle` user manually. 
+
+    # useradd -u 1000 -g 1000 oracle 
+
+Run the following commands as root:
+
+    # mkdir -p /scratch/DockerVolume/SOAVolume/
+    # chown 1000:1000 /scratch/DockerVolume/SOAVolume/
+    # chmod 700 /scratch/DockerVolume/SOAVolume/
+
+# Database
+
+You need to have a running database container or a database running on any machine. The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
+
+Please refer [README.md](https://github.com/oracle/docker-images/blob/master/OracleDatabase/README.md) under docker/OracleDatabase for details on how to build Oracle Database image.
+
+The DB image created through above step need to be retagged from `container-registry.oracle.com/oracle/database:12.2.0.1-ee`  to `oracle/database:12.2.0.1-ee` before continuing with next steps.
+
+$ docker tag container-registry.oracle.com/oracle/database:12.2.0.1-e  oracle/database:12.2.0.1-ee
```

Comment:
The Database image on Oracle Container Registry is different to the Database image you get if you build from the Dockerfile on GitHub. Has SOA been tested with both?